### PR TITLE
CI: exclude non-articles from style checks, and fix minor visual nuisances

### PR DESCRIPTION
**Pull request by TicClick**
unblocks https://github.com/ppy/osu-wiki/pull/8867

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -63,7 +63,10 @@ jobs:
 
       - name: run remark on changed files
         run: |
-          readarray -t REMARK_TARGET_FILES <<< $( echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ || true )
+          readarray -t REMARK_TARGET_FILES <<< $(
+            # ppy/osu-wiki#8867 -- disable style checks for regular Markdown files, such as README.md
+            echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ | grep -E -v '^[[:upper:]]+\.md$' || true
```
Comment (Walavouchey) [inline .github/workflows/continuous-integration.yml:68]: what kind of wack-ass syntax is this? wouldn't something like this work?

```suggestion
            echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ | grep -E -v '^[A-Z-]+\.md$' || true
```

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -63,7 +63,10 @@ jobs:
 
       - name: run remark on changed files
         run: |
-          readarray -t REMARK_TARGET_FILES <<< $( echo "${{ env.INTERESTING_FILES }}" | grep -e .md$ || true )
+          readarray -t REMARK_TARGET_FILES <<< $(
+            # ppy/osu-wiki#8867 -- disable style checks for regular Markdown files, such as README.md
```
Comment (Walavouchey) [inline .github/workflows/continuous-integration.yml:67]:
```suggestion
            # ppy/osu-wiki#8867 -- disable style checks for non-article Markdown files, such as README.md
```

File: run-checks.sh [hunk 3]
```diff
@@ -57,7 +57,8 @@ function main() {
     exit 0
   fi
 
-  interesting_articles=$( echo "${interesting_files}" | grep -e ^wiki/ -e ^news/ | grep .md$ )
+  # ppy/osu-wiki#8867 -- disable style checks for regular Markdown files, such as README.md
```
Comment (Walavouchey) [inline run-checks.sh:60]:
```suggestion
  # ppy/osu-wiki#8867 -- disable style checks for non-article Markdown files, such as README.md
```

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (TicClick) [reply .github/workflows/continuous-integration.yml:68]: https://www.gnu.org/software/grep/manual/html_node/Character-Classes-and-Bracket-Expressions.html not PCRE, but it works (: can change for readability if you insist

```
/home/user % grep --version
grep (GNU grep) 3.7
--- snip ---
/home/user % echo -en 'a\nb\nDONALD-DUCK\nen.md\nCONTRIBUTING.md' | grep -Ev '^[[:upper:]]+\.md$'
a
b
DONALD-DUCK
en.md
```

Comment (Walavouchey) [reply .github/workflows/continuous-integration.yml]: as is it wouldn't exclude [`TEMPALTE-SERIES.md`](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE-SERIES.md) i'm pretty sure, so may want to add the dash

**Pull request merged**